tensorflow_datasets/core/features/video_feature.py

```diff
@@ -197,26 +185,80 @@ def to_json_content(self) -> Json:
         'ffmpeg_extra_args': self._extra_ffmpeg_args
     }
 
+  def _generate_video_html(self, images, framerate=24, encoding_format='mp4') -> str:
+    """Converts sequence of images into video string."""
+    imgs = len(str(len(images)))+1  # Find number of digits in len to give names.
+
+    with tempfile.TemporaryDirectory() as video_dir:
```

Comment:
Is there a way to test that `ffmpeg` is installed before writing on disk ? (e.g. run a dummy `ffmpeg --version` command) As this isn't required for GIF generation.